Versionsmigration/4.1-5.4/xplangml-migration-41-54.halex.alignment.xml

```diff
@@ -5386,11 +5386,11 @@ else return besondereZweckbestimmung
         </target>
         <complexParameter name="script">
             <core:text xmlns:core="http://www.esdi-humboldt.eu/hale/core" xml:space="preserve">
-if (besondereZweckbestimmung in ["28000", "28001", "28002", "28003", "28004"]) {&#xD;
```

Comment:
Might make sense to check if besondereZweckbestimmung occurrences would also need to be adapted for other cases in the mapping.

Reply:
Found this also in the feature type "FP_VerEntsorgung". I'll adjust this.